source/_integrations/modbus.markdown

```diff
@@ -454,6 +466,54 @@ climates:
       required: false
       type: string
       default: C
+    hvac_mode_register:
+      description: Definition of a register holding and controlling an HVAC mode
+      required: false
+      type: [map]
+      keys:
+        address: 
+          description: The address of the HVAC mode register.
+          required: true
+          type: integer
+        values:
+          description: A mapping between the register values and HVAC modes
+          required: true
+          type: [map]
+          keys:
+            "off":
+              description: The register value corresponding to HVAC Off mode.
+              required: false
+              type: integer
+            heat:
+              description: The register value corresponding to HVAC Heat mode.
+              required: false
+              type: integer
+            cool:
+              description: The register value corresponding to HVAC Cool mode.
+              required: false
+              type: integer
+            auto:
+              description: The register value corresponding to HVAC Auto mode.
+              required: false
+              type: integer
+            dry:
+              description: The register value corresponding to HVAC Dry mode.
+              required: false
+              type: integer
+            fan_only:
+              description: The register value corresponding to HVAC Fan only mode.
+              required: false
+              type: integer
+            heat_cool:
+              description: The register value corresponding to HVAC Heat/Cool mode.
+              required: false
+              type: integer
+    hvac_onoff_register:
+      description: Definition of a register holding and controlling the On/Off state of the climate device.
+        A zero value in this register means that the device is in off state, otherwise it is on. When writing
```

Comment:
It would be more logical to use True/False.

Reply:
I don't understand what you mean. The address of the register is an integer. The value being read or written is also integer.

Reply:
Yes but the value can only be 0 or non-zero, which is the same as 0 == False and otherwise it is True. Using an integer makes you believe that non-zero values have different meanings.

Reply:
The configurable value is only an address. I've changed the description, hope it's not misleading now.